modules/roxctl-image-scan-command-options.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * cli/image-scanning-by-using-the-roxctl-cli.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="roxctl-image-scan-command-options_{context}"]
+= roxctl image scan command options
+
+The `roxctl image scan` command supports the following options:
+
+[cols="1m,2",options="header"]
```

Comment:
Can we fix the spacing in the left column of the table? It looks terrible in the preview.

Reply:
It is the last row in the table that is causing the spacing issue:
![Screenshot from 2023-10-31 13-00-37](https://github.com/openshift/openshift-docs/assets/61876435/21a12683-20d2-4cda-9b2e-52295c611026)
@dcaravel For the description related to `--row-jsonpath-expressions string`, can we render the default value in the description colomn using "\" for better readability?

Reply:
TL;DR; That will not work in this case, a failure will occur if any whitespace is in the value.  Perhaps remove the default value and add a note along the lines of: "For more details, run `roxctl image scan --help`" .

I tested out various values in `bash` (which looks like our docs assume is the shell being used):

Working example:
```
roxctl image scan --image "nginx:1.14.0" -o csv --row-jsonpath-expressions "{result.vulnerabilities.#.componentName,result.vulnerabilities.#.componentVersion,result.vulnerabilities.#.cveId}"

COMPONENT,VERSION,CVE,SEVERITY,LINK
apt,1.4.8,CVE-2019-3462
apt,1.4.8,CVE-2020-3810
```

Adding whitespace after the last closing bracket `} `, no data is returned:
```
roxctl image scan --image "nginx:1.14.0" -o csv --row-jsonpath-expressions "{result.vulnerabilities.#.componentName,result.vulnerabilities.#.componentVersion,result.vulnerabilities.#.cveId} "

COMPONENT,VERSION,CVE,SEVERITY,LINK

```
Adding a whitespace before the last closing bracket ` }`, error returned:

```
rctl image scan --image "nginx:1.14.0" -o csv --row-jsonpath-expressions "{result.vulnerabilities.#.componentName,result.vulnerabilities.#.componentVersion,result.vulnerabilities.#.cveId }"

ERROR:	scaning image failed after 3 retries: could not print CVE summary: invariant violation: jagged array found: yielded values within each array are not matching; expected each array to hold 282 elements but found an array with 1 elements at array index 2
```

Reply:
OK, let us remove the example entirely. 
I will go with: 
"Use the JSON path expressions to create rows from the JSON object. For more details, run the `roxctl image scan --help` command."